src/test/java/seedu/duke/helper/ParserTest.java

```diff
@@ -25,20 +26,20 @@ void printDetails(TestInfo testInfo) {
     void validateMedicineTest() {
         String[] userInputArray = {"paracetamol", "500", "2023-02-02", "Headaches", "500"};
         String userInput = "paracetamol,500,2023-02-02,Headaches,500";
-        assertEquals(true, Parser.validateMedicine(userInputArray));
+        assertTrue(Validator.validateMedicine(userInputArray));
     }
 
     @Test
     @DisplayName("parseAddMedicine method in Parser Class")
-    void parseAddMedicineTest() {
+    void parseAddMedicineTest() throws HalpmiException {
```

Comment:
Why does your test method throw an exception? When is it thrown? Because i don't see any exception being thrown?